### PR TITLE
modified for firebase suggested events for android

### PR DESCRIPTION
**Pull request by bijoshtj**

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -1,32 +1,85 @@
...
+    this.setScreen = function (name) {
+      NATIVE.plugins.sendEvent("FirebasePlugin", "setScreen", JSON.stringify({
+        name: name
+      }));
+    };
 });
 
 exports = new FireBase();
+exports.EVENTS = EVENTS;
+exports.PARAMS = PARAMS;
```
Comment (rampr) [inline js/index.js]: will this be available as Firebase.EVENTS ?

File: android/FirebasePlugin.java [hunk 2]
```diff
@@ -14,6 +14,8 @@
 import android.support.v4.app.FragmentActivity;
 
 import com.google.firebase.analytics.FirebaseAnalytics;
+import com.google.firebase.analytics.FirebaseAnalytics.Event;
+import com.google.firebase.analytics.FirebaseAnalytics.Param;
```
Comment (rampr) [inline android/FirebasePlugin.java]: are these imports needed now?

Comment (rampr) [inline android/FirebasePlugin.java]: I guess Param alone is sufficient

File: js/index.js [hunk 3]
```diff
@@ -1,32 +1,85 @@
...
+      }
+      NATIVE.plugins.sendEvent("FirebasePlugin", "setUserData", JSON.stringify(data));
+    };
+
+    this.logEvent = function (e_name, params) {
+      if (e_name in fire_events) {
+        params = _.pick(params, fire_events[e_name]);
+      }
+
+      console.log('FIRLOG', e_name, JSON.stringify(params));
```
Comment (rampr) [inline js/index.js]: you were going to remove the log ?

File: js/index.js (hunk 1 above)
Comment (bijoshtj) [inline js/index.js]: yo!

File: android/FirebasePlugin.java (hunk 2 above)
Comment (bijoshtj) [inline android/FirebasePlugin.java]: its needed for sending app_open event

**Pull request closed without merging**